src/parser/jobs/ast/ArcanaUndrawUsage.js

```diff
@@ -0,0 +1,97 @@
+import React, {Fragment} from 'react'
+
+import ACTIONS from 'data/ACTIONS'
+// import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+
+import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import {ActionLink} from 'components/ui/DbLink'
+
+const OGCD_ARCANA_REMOVAL = [
+	ACTIONS.UNDRAW_SPREAD.id,
+	ACTIONS.EMPTY_ROAD.id,
+	ACTIONS.UNDRAW.id,
+]
+
+export default class ArcanaUndrawUsage extends Module {
+	static handle = 'arcanaundraws'
+	static dependencies = [
+		'castTime',
+		'gcd',
+		'suggestions',
+	]
+
+
+	constructor(...args) {
+		super(...args)
+
+		this.addHook('cast', {by: 'player'}, this._onCast)
+		this.addHook('complete', this._onComplete)
+		this._badUndraws = []
+	}
+
+
+	_onCast(event) {
+
+		this._saveIfBadUndraw(event)
+
+	}
+
+	_saveIfBadUndraw(event) {
+		const actionId = event.ability.guid
+
+		// It's a OGCD Arcana Undraw. GET IT.
+		if (OGCD_ARCANA_REMOVAL.includes(actionId)) {
+			this._badUndraws.push(event)
+		} 
+	}
+
+	_onComplete() {
+		
+		const badUndraws = this._badUndraws
+			console.warn(this._badUndraws, 'undraws')
+		if (badUndraws.length) {
+			this.suggestions.add(new Suggestion({
+				icon: 'https://secure.xivdb.com/img/game/003000/003108.png', // Undraw action
+				content: <Fragment>
```

Comment:
This content block is too big, if we're gonna keep this suggestion, will need to move the macros into some other format.